### PR TITLE
use litelement properly for debugmenu

### PR DESCRIPTION
**Pull request by rom1504**

### CODE REVIEW

File: lib/debugmenu.js [hunk 1]
```diff
@@ -41,182 +49,50 @@ class DebugMenu extends LitElement {
...
-            this.setEntryVisible(this.targetBlockEntry, false)
-          }
-        }
-        this.bot.on('move', this.updateListener)
-      } else if (this.updateListener) {
-        this.bot.removeListener('move', this.updateListener)
-        this.updateListener = null
-      }
+    if (!this.isOpen) {
+      return html``
```
Comment (Moondarker) [inline lib/debugmenu.js:60]: We can save at least 1+ function call if this gets prevented in shouldUpdate () instead

File: lib/debugmenu.js [hunk 2]
```diff
@@ -41,182 +49,50 @@ class DebugMenu extends LitElement {
 
   static get properties () {
     return {
-      debugEntries: { type: Object },
       isOpen: { type: Boolean },
-      cursorBlock: { type: Object }
+      cursorBlock: { type: Object },
```
Comment (Moondarker) [inline lib/debugmenu.js:53]: The entire update sequence holds on changing cursorBlock, nothing updates if you look into the sky

Comment (rom1504) [reply lib/debugmenu.js:54]: indeed, fixed

File: lib/debugmenu.js (hunk 1 above)
Comment (rom1504) [reply lib/debugmenu.js:71]: theoritically this makes sense, but I tried it and overriding shouldUpdate to return this.isOpen results in the f3 key not being caught, I'm not 100% sure as to why, so let's keep it like this

Comment (Moondarker) [reply lib/debugmenu.js:71]: Probably because you set event listener in `firstUpdated ()` which wouldn't run unless menu is open at that moment

**Pull request merged**